View/Helper/TagHelper.php

```diff
@@ -56,6 +56,8 @@ public function input($field, $identifier, $options=array()) {
 			$tagid .= ucfirst($f);
 		}
 		$this->Js->buffer("handleTags('$tagid');");
+		
+		echo $this->Html->scriptBlock('var maxTagCharLength = ' . Configure::read('Tag.tag_name_length') . ';', [ 'inline' => true ]);
```

Comment:
This fix makes this plugin not quite standalone, as the configuration is only found in core.

We should also set a default number here if configuration is not found.